### PR TITLE
[stable/mediawiki] allow use of external database

### PR DESCRIPTION
**Pull request by tompizmor**
Since `bitnami/mediawiki:1.29.2-r1` it is possible to configure MediaWiki to use an already existing database.
This PR makes the required changes in the chart to create the database using the MariaDB chart and use it for MediaWiki.

### CODE REVIEW

File: stable/mediawiki/templates/deployment.yaml [hunk 1]
```diff
@@ -19,11 +19,26 @@ spec:
...
         - name: MARIADB_HOST
           value: {{ template "mediawiki.mariadb.fullname" . }}
         - name: MARIADB_PORT_NUMBER
           value: "3306"
-        - name: MARIADB_PASSWORD
+        - name: MEDIAWIKI_DATABASE_NAME
+          value: {{ default "" .Values.mariadb.mariadbDatabase | quote }}
+        - name: MEDIAWIKI_DATABASE_USER
+          value: {{ default "" .Values.mariadb.mariadbUser | quote }}
+        - name: WORDPRESS_DATABASE_PASSWORD
```
Comment (sozercan) [inline stable/mediawiki/templates/deployment.yaml]: @tompizmor this needs to be `MEDIAWIKI_DATABASE_PASSWORD`  probably :)

Comment (tompizmor) [reply stable/mediawiki/templates/deployment.yaml]: yes!

**Comment by tompizmor** [comment]
@prydonius can you recheck this?

**Review by sameersbn — CHANGES_REQUESTED** [review]
@tompizmor  my comments from https://github.com/kubernetes/charts/pull/2815 apply here too.

**Review by sameersbn — APPROVED** [review]

**Pull request merged**